packages/scaffold-config/src/ct-detect-third-party.ts

```diff
@@ -18,54 +19,41 @@ const DependencyArraySchema = z.array(DependencySchema)
 
 const BundlerSchema = z.enum(['webpack', 'vite'])
 
+const thirdPartyDefinitionPrefixes = {
```

Comment:
Unfortunately duplication, don't have a great way to share code between here and `npm/webpack-dev-server`.